prfiesta/output.py

```diff
@@ -24,7 +24,7 @@ def output_frame(
         timestamp = datetime.now()
 
     if not output_name:
-        output_name = f"export.{timestamp.strftime('%Y-%m-%d_%H:%M:%S')}.{output_type}"
+        output_name = f"export.{timestamp.strftime('%Y%m%d%H%M%S')}.{output_type}"
```

Comment:
We should add a unit test which basically says: Given the user has not provided an output name + they are running in windows, ensure that the default output name never includes any of the disallowed characters

This should protect us from any regression introduced in the future :smile: 

It would also be great if we added a check when the user does provide a custom output name + they are on windows then the custom output name does not include any of those same characters (potentially emit a warning when this happens)